src/daft-logical-plan/src/builder/resolve_expr.rs

```diff
@@ -270,8 +270,6 @@ fn convert_udfs_to_map_groups(expr: &ExprRef) -> ExprRef {
 /// where they are not allowed, and resolves struct accessors and wildcards.
 #[derive(Default, TypedBuilder)]
 pub struct ExprResolver<'a> {
-    #[builder(default)]
-    allow_actor_pool_udf: bool,
```

Comment:
Will actor pool UDFs work everywhere now? I believe there are other ops in which we do not split yet and thus won't work, such as agg.

Reply:
Wait do we allow arbitrary expressions in aggs?

Reply:
Ok I'll just remove from filters for now, but we should figure out how to generalize this